main.py

```diff
@@ -0,0 +1,17 @@
+from selenium import webdriver
+import json
+import constants
+
+def main():
+    with open(constants.JSON_PATH) as f:
+        fyi_links = json.load(f)
+    driver = webdriver.Chrome()
+    driver.get(fyi_links[constants.NETFLIX_SALARY_LINK])
```

Comment:
Can we abstract this?

Reply:
Depends... it is good form to keep constants out of logic files... in java, unsure if that is still a best practice in python. Would still want to keep this data in JSON format though in prep for scaling.